bofire/strategies/predictives/enting.py

```diff
@@ -0,0 +1,370 @@
+import warnings
+from typing import List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
+
+try:
+    import entmoot.constraints as entconstr  # type: ignore
+    import pyomo.environ as pyo  # type: ignore
+    from entmoot.models.enting import Enting  # type: ignore
+    from entmoot.optimizers.pyomo_opt import PyomoOptimizer  # type: ignore
+    from entmoot.problem_config import ProblemConfig  # type: ignore
+except ImportError:
+    warnings.warn("entmoot not installed, BoFire's `EntingStrategy` cannot be used.")
+
+from typing import Union
+
+from pydantic import PositiveInt
+
+import bofire.data_models.strategies.api as data_models
+from bofire.data_models.constraints.api import (
+    LinearEqualityConstraint,
+    LinearInequalityConstraint,
+    NChooseKConstraint,
+)
+from bofire.data_models.domain.api import Domain
+from bofire.data_models.features.api import (
+    AnyOutput,
+    CategoricalInput,
+    ContinuousInput,
+    DiscreteInput,
+)
+from bofire.data_models.objectives.api import MaximizeObjective, MinimizeObjective
+from bofire.strategies.predictives.predictive import PredictiveStrategy
+
+
+def domain_to_problem_config(
+    domain: Domain, seed: Optional[int] = None
+) -> Tuple[ProblemConfig, pyo.ConcreteModel]:
```

Comment:
I think having this import specific signature is a problem. Putting it in `""` should solve it.

```suggestion
) -> Tuple["ProblemConfig", "pyo.ConcreteModel"]:
```

Reply:
I will change this for all signatures.